maps/src/main/java/org/odk/collect/maps/MapFragmentDelegate.kt

```diff
@@ -0,0 +1,29 @@
+package org.odk.collect.maps
+
+import android.os.Bundle
+import org.odk.collect.shared.settings.Settings
+import org.odk.collect.shared.settings.Settings.OnSettingChangeListener
+import java.util.function.Consumer
+
+class MapFragmentDelegate(
+    private val configurator: MapConfigurator,
+    private val settings: Settings,
+    private val onConfigChanged: Consumer<Bundle>,
+) :
+    OnSettingChangeListener {
```

Comment:
We usually do not add a new line in places like here I'm surprised it's even ok according to ktlint.